priv/templates/phx.gen.auth/auth_test.exs

```diff
@@ -152,6 +157,72 @@ defmodule <%= inspect auth_module %>Test do
       refute get_session(conn, :<%= schema.singular %>_token)
       refute conn.assigns.current_scope
     end
+
+    defp generate_offset_<%= schema.singular %>_session_token(<%= schema.singular %>, amount_to_add, unit) do
+      token = <%= inspect context.alias %>.generate_<%= schema.singular %>_session_token(<%= schema.singular %>)
+      {token, offset_<%= schema.singular %>_token(token, amount_to_add, unit)}
+    end
+
+    defp offset_<%= schema.singular %>_token(token, amount_to_add, unit) do
+      dt = DateTime.add(DateTime.utc_now(), amount_to_add, unit)
+      query = from(ut in <%= inspect schema.alias %>Token, where: ut.token == ^token)
+      {1, nil} = Repo.update_all(query, set: [inserted_at: dt, refreshed_at: dt])
+      {<%= schema.singular %>_token, _} = <%= inspect context.alias %>.get_<%= schema.singular %>_auth_by_session_token(token)
+      <%= schema.singular %>_token
+    end
+
+    test "does not refresh the session token after only a few hours", %{conn: conn, <%= schema.singular %>: <%= schema.singular %>} do
```

Comment:
Try to squeeze this into a previous test, possibly by checking that the token in the session is still the same.

Reply:
Well, it seems you already do so once you changed line 147, so we can nuke this test.